src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
```

Comment:
`else if` redundant when in prev block `return`
```suggestion
  }
  
  if (days >= 3) {
```